### PR TITLE
Njray/sharepoint2

### PR DESCRIPTION
**Pull request by njray**

### CODE REVIEW

**Comment by bennage** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference-architectures/SharePoint/index.md](https://github.com/njray/architecture-center/blob/njray/sharepoint2/docs/reference-architectures/SharePoint/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/sharepoint/index?branch=pr-en-us-174) | [Details](#user-content-docs%2Freference-architectures%2FSharePoint%2Findex.md)

<a name="docs%2Freference-architectures%2FSharePoint%2Findex.md"></a>
### [docs/reference-architectures/SharePoint/index.md](https://github.com/njray/architecture-center/blob/njray/sharepoint2/docs/reference-architectures/SharePoint/index.md)
 - **Line 328**: **[Warning]** Invalid file link:(~/reference-architectures/SharePoint/CLI).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C21%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201706212244401846-174%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by njray** [comment]
index.md for larry/sharepoint containing latest deploy steps

**Comment by lbrader** [comment]
Nanette - appears there a issue from Bennage bot

File: docs/reference-architectures/SharePoint/index.md [hunk 1]
```diff
@@ -0,0 +1,553 @@
+---
```
Comment (MikeWasson) [inline docs/reference-architectures/SharePoint/index.md]: path should be all lowercase

docs/reference-architectures/sharepoint/index.md

File: docs/reference-architectures/SharePoint/index.md [hunk 2]
```diff
@@ -0,0 +1,553 @@
...
+[sql-maintenance]: https://technet.microsoft.com/en-us/library/hh292622(v=office.16).aspx
+[storage-planning]: https://technet.microsoft.com/en-us/library/cc298801(v=office.16).aspx
+[storage-volumes]: https://technet.microsoft.com/en-us/library/cc263445.aspx
+[subnet]: /azure/virtual-network/virtual-network-manage-subnet
+[technet-wiki]: https://social.technet.microsoft.com/wiki/contents/articles/12438.sharepoint-community-best-practices.aspx
+[tempdb]: https://technet.microsoft.com/en-us/library/ms175527(v=sql.105).aspx
+[vm-sizes]: /azure/virtual-machines/virtual-machines-windows-sizes
+[vpn]: /azure/vpn-gateway/vpn-gateway-about-vpngateways
+[windows-vm-workload]: /azure/architecture/reference-architectures/virtual-machines-windows/
+[0]: ./images/Sharepoint-2016-RA-diagram.png "Highly available SharePoint Server 2016 farm in Azure."
```
Comment (MikeWasson) [inline docs/reference-architectures/SharePoint/index.md]: Wrong image path

**Comment by MikeWasson** [comment]
The latest doc should be in the [sharepoint-ra branch](https://github.com/mspnp/architecture-center/blob/sharepoint-ra/docs/reference-architectures/sharepoint/index.md)

At least, that's the one I made edits on.

**Comment by MikeWasson** [comment]
Dup of #176

**Pull request closed without merging**